### ISSUE TITLE
Error: getStatus for non-existing user does not return false, results in PATCH

### ISSUE THREAD

**Issue by ml-eds** [danielstieber/SubscribeToMailchimp#10]
When querying member info for a non-existent email, Mailchimp API returns status `404`. The getStatus method of the module returns this value, which wrongly triggers a PATCH operation in subscribe method at [line 54](https://github.com/danielstieber/SubscribeToMailchimp/blob/master/SubscribeToMailchimp.module#L54) of curent master.

Perhaps it would be a good idea to explicitly check `404`? I suggest

```
            if($status !== false && $status !== 404) {
           .....
```

I can do a PR, if you want me to.

**Comment by ml-eds** [comment]
Or perhaps it would be better to handle the 404 response within getStatus and let this method consistently return false when an email does not exist.

**Comment by danielstieber** [comment]
That's a big problem, thanks a lot. I make a PR for #9 and #10 quickly.

**Comment by danielstieber** [comment]
fixed in v0.0.6
